bin/mc-send-to-console

```diff
@@ -12,8 +12,13 @@ if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
   exit 1
 fi
 
+distro=$(cat /etc/os-release | grep -E "^ID=" | cut -d= -f2 | sed -e 's/"//g')
```

Comment:
Should this be a global variable?

Reply:
Definitely no, I'm not very into bash scripting. Thanks for looking, I'll fix in a bit :)

Reply:
It looks like there's already global variable called `distro` so we can just use it. I removed mine completely.

Reply:
No, there's not a global variable. Instead there's a helper function used as:

```
  distro=$(getDistro)
```

however, `mc-send-to-console` doesn't currently import start-utils, which would need this at the top

```
. "/start-utils"
```

Reply:
That's weird. I rebuilt the container and the command seems to work just fine.
```
root@7b2c0b045c7c:/data# cat /usr/local/bin/mc-send-to-console 
#!/bin/bash

: "${CONSOLE_IN_NAMED_PIPE:=/tmp/minecraft-console-in}"

if [ $# = 0 ]; then
  echo "ERROR: pass console commands as arguments"
  exit 1
fi

if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
  echo "ERROR: named pipe ${CONSOLE_IN_NAMED_PIPE} is missing"
  exit 1
fi

if [ "$(id -u)" = 0 ]; then
  if [[ $distro == alpine ]]; then
    exec su-exec minecraft bash -c "echo $* > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
  else
    exec gosu minecraft bash -c "echo $* > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
  fi
else
  echo "$@" >"${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}"
fi
root@7b2c0b045c7c:/data# mc-send-to-console op CDFN
root@7b2c0b045c7c:/data# 
```

Reply:
Since `distro` is unset, it'll be an empty string. So it'll be going in the `else` block, using `gosu` without any errors on a debian based image.

Reply:
Oh... I forgot that when you won't set --build-args it uses debian by default so that would be why :facepalm: I'll re-test with alpine based image, sorry for that :sweat_smile:

Reply:
...fyi if you're doing a docker build with the default, then it's building a debian image. Add the build args `BASE_IMAGE=openjdk:8-jdk-alpine3.8; DISTRO=alpine` to build a java8 alpine image.

Reply:
Should be sorted now :)